hack/verify-codegen.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
Should this script be running somewhere in our CI? We should add it in a follow-up PR

Reply:
yes I think so, created a PR https://github.com/kubernetes/cloud-provider-aws/pull/153